ote/src/cljc/ote/time.cljc

```diff
@@ -31,28 +31,37 @@
 (s/def ::specql-data-types/time #(instance? Time %))
 
 #?(:cljs
-(defn format-timestamp-for-ui [time]
-  (if  (nil? time)
-    " " ;: if nil - print empty string
-    (->> time
+   (defn format-timestamp-for-ui [timestamp]
+     (if (nil? timestamp)
+       " " ;: if nil - print empty string
+       (->>
+         (if (string? timestamp)
+           (coerce/from-string timestamp)
```

Comment:
This function should NOT be called with a string parameter. I think we should fix erroneus call sites instead.

Reply:
Yes, I can do that. Will change the original "timestamp" parameter name too, because can be misleading.